### ISSUE TITLE
When using alsa output, there's an error.

### ISSUE THREAD

**Issue by nemonein** [anonbeat/guayadeque#18]
I'm using guayadeque 0.4.1.r2025.757e8ff~xenial-1 (PPA version) on Ubuntu 16.04.
I set the output device as ALSA, hw:1,0. (This setting works fine with the Audacious player.), and tried to play songs from my library.

There's an error like this, on the terminal where I started guayadeque.
```Error: ee: Failed load of file 'file:///media/Music-Nfs/Main/Jazz/Charles Mingus/1957 - East Coasting/02 - East Coasting.flac'

Error: ee: Failed load of file 'file:///media/Music-Nfs/Main/Jazz/Charles Mingus/1957 - East Coasting/04 - Celia.flac'````

It plays well if I choose Pulse as an output device.
I'd be glad if you could find what's wrong.

Regards,
JH. Park.

**Comment by anonbeat** [comment]
Can you send me to my email the rest of the output when fails ? 

Thanks

**Comment by anonbeat** [comment]
Also be aware that alsa only allow one playback stream for device unless you configure dmix. Be sure you disable crossfading with alsa and no other program is using it.

**Comment by nemonein** [comment]
@anonbeat Thanks, I think crossfader was the reason.
I disabled the cross fader(Fade-out length to 0.0), it works fine.

By the way, I hope Guyadeque would report the error while there's something wrong with alsa setting.
For example,if I set the wrong device on the setting. hw:1,0 is the correct one, but I set to hw:2,0, Guyadeque would not report anything but fail to load the music files. (and remove the song from the play list.)

Audacious reports something like this, if I make the same mistake.
'''ALSA error: snd_mixer_attach failed'''

Anyway, thank you for your effort. I've been using Guyadeque for a long time.
I have a few more suggestion for it, and I'll post it someday.

Regards,
JH.
